libraries/utilities/generate_clusters_from_pool.py

```diff
@@ -407,8 +407,8 @@ def write_config(config, pool_file, use_docker, sg_windows, sg_accel_windows,
         if sg_windows:
             sg_dict = {
                 "sync_gateways:vars": {
-                    "ansible_user": "Administrator",
-                    "ansible_password": "Membase123",
+                    "ansible_user": "FakeUser",
```

Comment:
I'm confused, how is it different from the previous value? This is still a hardcoded value and will never be overwritten for any other combination of Username and password. If the values are not used anywhere, then we shouldn't even write them in config in first place.

Reply:
Hemant, can you check previous code which is above this ? generate clusters per pool is used to generate clusters. we do not pass parameters of username and password when we run this script. we replace with fakeuser/fakepassword with actual username/password using 'sed' command on jenkins config and run this.